August2020 updates (#13)
b2io/handbook: code-of-conduct.md

Openness, honesty, and empathy are the building blocks of our culture. Being excellent to each other is the ultimate goal, and we recognize that setting clear guidelines around what constitutes "being excellent" can be helpful and reassuring for all team members. With that, we've put together this Code of Conduct. As with any portion of this handbook, if you feel something is missing from this section, let's work together to update it.

## Dress Code

> 

Reviewer: linter stuff again.

### How to Report an Issue

If you, as a member of our team, feel uncomfortable for any reason you should feel empowered to speak to any other team member about the situation. If the issue stems from a specific person on the team and you feel comfortable talking through it with them directly, you should. If you do not feel comfortable doing that, you can bring your concerns to another member of the team and ask for their help in resolving the matter.

### If Someone Reports an Issue to You

We are a very small team, and currently have no formal HR or Ethics group. Because of this, we rely on ourselves and each other to ensure that our workplace is safe and welcoming.

In any instance where a team member comes to you with an issue, first ask the team member if they would like you to take action. If the answer is yes:

- Commit to assisting in resolving their issue, or identify a person who can help them
- Channel your childhood [peer mediation techniques](http://www.sese.org/wp-content/uploads/2013/08/Peer-Mediation-guide.pdf)
- Act as an unbiased third party
- Strive for confidentiality as much as possible

> We don't have much of a formal system, it's something we are continuing to create and refine as a team. If you have ideas, do not hesitate to share them.

Reviewer 2: 💭 Possible re-word

```suggestion
Because we don't have a formal system, we're often refining this process as a team. If you have ideas, please share them.
```

- Inappropriate physical contact
- Unwelcome sexual attention
- Reporting for work under the influence of illegal drugs
- Bringing a weapon into the workplace

## Confidentiality

Base Two employees must adhere to any client's confidentiality guidelines, as well as use discretion when relaying potential client work or internal communications in an external way. If there is any question as to what sort of information is confidential, it is best to error on the side of caution or to ask a colleague/supervisor.

## Equal Opportunity Employment

Base Two is an equal opportunity employer and does not discriminate on the basis of race, ethnicity, religion, creed, sex, gender, sexual orientation, age, marital status, pregnancy or parental status, veteran status, national origin or ancestry, physical or mental disability, medical condition, family medical history or genetic information, military service, or other non-merit based factor We do not tolerate discrimination in including but not limited to including recruitment and hiring practices, appraisal systems, promotions, training, and career development programs.

## Reasonable and Fair Accommodation for Persons with Disabilities

> Base TWo will provide reasonable and fair accommodation to individuals with a physical or mental disability. Criteria being that accommodation would not impose an undue hardship on Base Two or other employees, and would enable the individual to apply for employment or perform essential functions of their position. It is the responsibility of every applicant or employee who requires accommodation to make his or her needs known, and understand that Base Two will engage in this accommodation process in good faith. For accommodations related to completing a job application, please contact [info@base2.io](mailto:jobs@base2.io).

Reviewer 3: Typo fix
```suggestion
Base Two will provide reasonable and fair accommodation to individuals with a physical or mental disability. Criteria being that accommodation would not impose an undue hardship on Base Two or other employees, and would enable the individual to apply for employment or perform essential functions of their position. It is the responsibility of every applicant or employee who requires accommodation to make his or her needs known, and understand that Base Two will engage in this accommodation process in good faith. For accommodations related to completing a job application, please contact [info@base2.io](mailto:jobs@base2.io).
```

## At Will Employment

> This handbook serves as employee guidlines, not as an employment contract.

Reviewer: In doing some handbook research, this was noted and I thought it's worth outlining in writing.

> > should we include an At-Will statement here?

Reviewer 2: I'm not sure. Has anyone seen an example that they like?
